jubilant/statustypes.py

```diff
@@ -174,7 +202,7 @@ class AppStatus:
     address: str = ''
     life: str = ''
     app_status: StatusInfo = dataclasses.field(default_factory=StatusInfo)
-    relations: dict[str, list[AppStatusRelation]] = dataclasses.field(default_factory=dict)  # type: ignore
+    relations: dict[str, list[AppStatusRelation | str]] = dataclasses.field(default_factory=dict)  # type: ignore
```

Comment:
Per discussion, let's use just `list[AppStatusRelation]` but only fill in the `interface` field.